elements/notification-container/notification-container.js

```diff
@@ -0,0 +1,96 @@
+import BulbsElement from 'bulbs-elements/bulbs-element';
+import NotificationDisplay from './notification-display';
+import invariant from 'invariant';
+import React, { PropTypes } from 'react';
+import { registerReactElement } from 'bulbs-elements/register';
+
+import './notification-container.scss';
+
+class NotificationContainer extends BulbsElement {
+
+  constructor (props) {
+    invariant(!!props.src, 'notification-container component requires a src');
+    super(props);
+    let idsFromStorage = JSON.parse(
+      global.localStorage.getItem('onion-local-notification-ids') || '[]'
+    );
+    this.state = {
+      next: null,
+      notifications_are_off: JSON.parse(
+        window.sessionStorage.getItem('onion-notifications-off') || 'false'
+      ),
+      local_notification_ids: idsFromStorage,
```

Comment:
I agree with the readability of the snake case, but according to conventions this is `localNotificationIds` :(